cdk/kraken/src/pypi.ts

```diff
@@ -89,9 +89,10 @@ export class PyPIPublishStack extends Stack {
       steps: [
         {
           name: 'Verify tag',
+          shell: 'bash',
           run: dedent`GIT_TAG=\${GITHUB_REF/refs\\/tags\\//}
           LIBRARY_VERSION=$(poetry version -s)
-          if [[ "$GIT_TAG" != LIBRARY_VERSION ]]; then exit 1; fi`,
+          if [[ "$GIT_TAG" != "$LIBRARY_VERSION" ]]; then echo "Tag ($GIT_TAG) does not match poetry version ($LIBRARY_VERSION)"; exit 1; fi`,
```

Comment:
Why is quotes around `LIBRARY_VERSION` needed? Is it because otherwise it is not a string and would not equal to `"$GIT_TAG"`?

Reply:
It's honestly just convention I picked up somewhere along the way. Based on this [SO](https://stackoverflow.com/questions/2981878/checking-for-environment-variables) link, it looks like that reason may be to ensure this check always works.